metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -268,40 +254,43 @@ final class BloopServers(
     scribe.info(s"bloop exit: $exitCode")
   }
 
-  private def bloopCommandLineIsInstalled(workspace: AbsolutePath): Boolean = {
+  private def bloopCommandLineIsInstalled(port: Int): Boolean = {
+    var socket: Socket = null
     try {
-      val output = Process(
-        List("python", embedded.bloopPy.toString(), "help"),
-        cwd = workspace.toFile
-      ).!!(ProcessLogger(_ => ()))
-      // NOTE: our BSP integration requires bloop 1.1 or higher so we ensure
-      // users are on an older version.
-      val isOldVersion =
```

Comment:
It makes sense to remove this bit of code, nobody should be using bloop 1.1.x :+1: